account-compression/README.md

```diff
@@ -1,13 +1,18 @@
-# Smart Contracts
+# Account Compression
 
-`gumdrop` will be managed by solana-labs 
+This on-chain program provides an interface for composing smart-contracts to create and use
+SPL ConcurrentMerkleTrees. The primary application of using SPL ConcurrentMerkleTrees is
+to make edits to off-chain data with on-chain verification. 
 
-`bubblegum`, `gumball-machine` and `gumdrop` are managed by metaplex-foundation
+Using this program requires an indexer to parse transaction information and write relevant information to an off-chain database.
```

Comment:
can you expand on what "relevant information" means here?

Reply:
This is a note for application devs. 

Relevant information refers to application-level information that gets hashed into the leaf.

This information has to be logged via cpi to `wrapper` in order for off-chain indexers to update properly.